examples/jsm/geometries/InstancedPointsGeometry.js

```diff
@@ -82,6 +82,26 @@ class InstancedPointsGeometry extends InstancedBufferGeometry {
 
 	}
 
+	setSizes( array ) {
```

Comment:
Instead of adding this buffer here using `setSizes()` could you move to example, and add the `material.pointSizeNode` property and let the user do their own setup? removing `useInstanceSize` too...

Reply:
Should I remove setSizes from InstancedPointsGeometry as well ( might be useful for WebGL cases?)

Reply:
I think it's better to remove it.

Reply:
I've addressed the issues mentioned above. If it's okay, I've also added a minor effect to pointColorNode to surface an additional node property to website viewers.